app/helpers/dashboard_helper.rb

```diff
@@ -6,4 +6,43 @@ def tab_class(type:)
       "active_tab" if type == :active
     end
   end
+
+  # We aren't storing a custody number at the moment,
+  # but it is part of the designs and should be there
+  # for demo purposes - just make one up.
+  def random_custody_number
+    "AW2015" + Array.new(5) { rand(9) }.join
+  end
+
+  def day_text(dr)
+    if dr.created_at < Time.zone.now.beginning_of_day
+      short_date_formatter(dr.created_at)
+    end
+  end
+
+  def arriving_text(dr)
+    if dr.created_at < Time.zone.now.beginning_of_day
+      # before today
+      "Arriving"
+    else
+      if dr.created_at < Time.zone.now.end_of_day
+        # today
+        "Arriving at"
```

Comment:
Perhaps it's better to take display text from `en.yml` locales file? e.g. change to `I18n.t('arriving_at')` etc